SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute.iOS/Distribute.cs

```diff
@@ -8,7 +8,19 @@ namespace Microsoft.Azure.Mobile.Distribute
     public static partial class Distribute
     {
         [Preserve]
-        public static Type BindingType => typeof(iOSDistribute);
+        public static Type BindingType
+        {
+            get
+            {
+
+// We cannot use Conditional Attributes as they only work for non-void methods and classes, not for properties.
+#if DEBUG
```

Comment:
To fix I suggest having an additional field variable to hold the BindingType value to return in property, and have a conditional void method to set it to null in debug or something like that.